pkg/handler/owncloud.go

```diff
@@ -276,17 +289,21 @@ func (s ownCloudSession) NewClient() *msgraph.GraphServiceRequestBuilder {
 
 func (s ownCloudSession) getUsers(userName string) ([]msgraph.User, error) {
 	if s.useGraphAPI {
+		s.log.V(6).Info("using graph api")
```

Comment:
Why `INFO` what about `DBG`?

Reply:
@micbar glauth uses the https://github.com/go-logr/logr interface. It has no log levels. Only `Error()` and `V(n).Info(...)`, where n is the verbosity. 6 is pretty verbose and in ocis glauth we map 6 to debug.